grafana-plugin/src/pages/schedules/Schedules.tsx

```diff
@@ -369,14 +369,14 @@ class SchedulesPage extends React.Component<SchedulesPageProps, SchedulesPageSta
             {item.on_call_now.map((user) => {
               return (
                 <PluginLink key={user.pk} query={{ page: 'users', id: user.pk }} className="table__email-content">
-                  <div className={cx('schedules__user-on-call')}>
-                    <div>
-                      <Avatar size="medium" src={user.avatar} />
-                    </div>
-                    <MatchMediaTooltip placement="top" content={user.username} maxWidth={TABLE_COLUMN_MAX_WIDTH}>
-                      <span className="table__email-content">{user.username}</span>
-                    </MatchMediaTooltip>
-                  </div>
+                  <HorizontalGroup>
+                    <TextEllipsisTooltip placement="top" content={user.username}>
+                      <Text type="secondary" className={cx('overflow-child')}>
+                        <Avatar size="small" src={user.avatar} />{' '}
```

Comment:
Changed from `medium` to `small` so that it's consistent with other places (see incident's users or the teams icon)